### PR TITLE
Amal_Joy

### PR DESCRIPTION
**Pull request by AmalJoy14**

### CODE REVIEW

File: participants/template.md
Comment (rajatsandeepsen) [inline participants/template.md:1]: dont delete this file

File: participants/README.md
Comment (rajatsandeepsen) [inline participants/README.md:1]: Rename this file to your-name-roll-no

**Review by rajatsandeepsen — CHANGES_REQUESTED** [review]
Cool ✨

**Comment by AmalJoy14** [comment]
Done!

**Comment by alvin-dennis** [comment]
@Milansuman u can merge this after ur final recheck

**Pull request merged**